roles/StackStorm.st2/tasks/auth.yml

```diff
@@ -31,6 +31,16 @@
   until: _task is succeeded
   when: ansible_facts.os_family == 'RedHat'
 
+# Use pip to install passlib as no python38-passlib yum package available. Still need python3-passlib
+# installed as well though from above task
+- name: auth | Install pip pre-reqs (RedHat)
+  become: yes
+  pip:
+     name: passlib
```

Comment:
Could we install the `passlib` via pip for all the OS flavors to get rid of this kind of mishmash:

https://github.com/StackStorm/ansible-st2/blob/02b5ffbde1ce35666bf0c106b80659270c9c6000/roles/StackStorm.st2/vars/debian.yml#L1-L3
https://github.com/StackStorm/ansible-st2/blob/02b5ffbde1ce35666bf0c106b80659270c9c6000/roles/StackStorm.st2/vars/redhat_8.yml#L1-L3
https://github.com/StackStorm/ansible-st2/blob/02b5ffbde1ce35666bf0c106b80659270c9c6000/roles/StackStorm.st2/vars/redhat.yml#L1-L3
https://github.com/StackStorm/ansible-st2/blob/02b5ffbde1ce35666bf0c106b80659270c9c6000/roles/StackStorm.st2/vars/debian_20.yml#L1-L3

Reply:
I think we'll still need to mess about a bit, as the pip executable differs. So we have to point to pip3 on EL8, but we'll need to point to different pip versions on some of the other installs. (I tried with just the default pip module and no executable name - but that failed :-( )
So we'll just be replacing the passlib variable with a pip variable.

Reply:
@armab We don't install default pip at moment for U20 as part of ansible install. So we'd need to add that in as well.
I'm not sure its going to make it any cleaner, just mean we'll have another pip on the box to perhaps get confused with....

Reply:
Either way looks like a rake.
Alright, thanks for checking.

LGTM :+1: